src/one_dragon/base/push/channel/push_plus.py

```
@@ -0,0 +1,153 @@
+                "topic": config.get('USER'),
+                "template": config.get('TEMPLATE'),
+                "channel": config.get('CHANNEL'),
+                "webhook": config.get('WEBHOOK'),
+                "callbackUrl": config.get('CALLBACKURL'),
+                "to": config.get('TO'),
+            }
+
+            # 发送请求
+            body = json.dumps(data).encode(encoding="utf-8")
+            headers = {"Content-Type": "application/json"}
+            response = requests.post(url=url, data=body, headers=headers).json()
```

Ruff (0.14.0)

Probable use of `requests` call without timeout

(S113)